content/docs/how-we-work/baseline/logging-details.md

```diff
@@ -0,0 +1,145 @@
+---
+title: Logging
+weight: 2
+description: We can't fix what we can't see. Logging is the most fundamental aspect of observability in how our applications run so that we fix a problem, even proactively. For this, we rely on something more sophisticated than simply logging into the database or syslog. Our tool of choice for this purpose is Sentry.
+---
+
+# Logging
+
+We can't fix what we can't see. Logging is the most fundamental aspect of observability in how our applications run so that we fix a problem, even proactively. For this, we rely on something more sophisticated than simply logging into the database or syslog. Our tool of choice for this purpose is [Sentry](https://axelerant.sentry.io/).
+
+Any application that you build must be configured to write logs to Sentry. This is fairly straightforward as Sentry has SDKs in a number of languages and frameworks. For Drupal, use the [Raven](https://www.drupal.org/project/raven) module to pull in the integration and also configure what should be logged.
```

Comment:
Since we have moved everything here, let's remove all the text following this from the main baseline page. That page should only have these paragraphs and make the word "Logging" in the first paragraph link to this page.